src/NServiceBus.Core/Routing/RoutingSettings.cs

```diff
@@ -72,7 +72,7 @@ public void RouteToEndpoint(Assembly assembly, string @namespace, string destina
         /// </summary>
         public void DoNotEnforceBestPractices()
         {
-            Settings.Set(RoutingFeature.EnforceBestPracticesSettingsKey, false);
+            Settings.Set(RoutingComponent.EnforceBestPracticesSettingsKey, false);
```

Comment:
interesting thought. Generally I don't see why that wouldn't work. But also, reading settings from the settings bag and applying it to the correct places is currently the most common approach regarding settings.